src/frontend/src/pages/CalendarPage/CalendarPage.tsx

```diff
@@ -82,6 +82,16 @@ const CalendarPage = () => {
     });
   };
 
+  const noUnconfirmedDRSButton = () => {
```

Comment:
you can name this NoUnconfirmedDRSButton = () => {
}
and then call it below like:
`<NoUnconfirmedDRSButton/>`

Reply:
The ActionsMenu takes a list of ButtonInfo, it won't let me pass in a component.

Reply:
I just pushed the latest change, can you let me know if that's ok?